src/App.tsx

```diff
@@ -1,25 +1,29 @@
 import React from 'react';
-import logo from './logo.svg';
+//import logo from './logo.svg';
 import './App.css';
 
+import { sampleList } from './sampleList';
+
+function ListDisplay() {
+  let items: JSX.Element[] = [];
```

Comment:
Since we only push to this array instead of reassign it, we should use `const` instead of `let`